tests/trx_generator/trx_provider.hpp

```diff
@@ -126,16 +126,14 @@ namespace eosio::testing {
       std::vector<logged_trx_data> _sent_trx_data;
    };
 
-   using fc::time_point;
-
    struct tps_test_stats {
       uint32_t          total_trxs = 0;
       uint32_t          trxs_left = 0;
       uint32_t          trxs_sent = 0;
-      time_point        start_time;
-      time_point        expected_end_time;
-      time_point        last_run;
-      time_point        next_run;
+      fc::time_point        start_time;
+      fc::time_point        expected_end_time;
+      fc::time_point        last_run;
+      fc::time_point        next_run;
```

Comment:
Consider lining these up. Suggestion, not a requirement.

```suggestion
      fc::time_point    start_time;
      fc::time_point    expected_end_time;
      fc::time_point    last_run;
      fc::time_point    next_run;
```

Reply:
Addressed. b0d2e602a3844cb68bdd1744415fa94abee410bb